sort/strings.go

```diff
@@ -41,7 +41,7 @@ func StringsByHex(x []string) {
 	sort.Slice(x, func(i, j int) bool {
 		valI, err := encoding.ASCIIHexToBytes.Encode([]byte(x[i]))
 		if err != nil {
-			panic(fmt.Sprintf("failed to sort strings by hex: %v", err))
+			panic(fmt.Sprintf("failed to encode ascii hex %s to bytes : %v", x[i], err))
```

Comment:
This data would have been validated before sorting, so this panic isn't likely?

Reply:
We could return an `error` and still sort the slice in-place.

Reply:
It will panic if we put an invalid hex value into the spec. So, on the first pack/unpack, it will panic. I don't think we have any other place where we validate tags.